com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsCommon.cs

```diff
@@ -103,11 +103,13 @@ public SettingsPanel()
         }
 
         #region IDebugDisplaySettingsData
-        UniversalRenderPipelineDebugDisplaySettings debugDisplaySettings => UniversalRenderPipelineDebugDisplaySettings.Instance;
-        public bool AreAnySettingsActive => debugDisplaySettings.AreAnySettingsActive;
-        public bool IsPostProcessingAllowed => debugDisplaySettings.IsPostProcessingAllowed;
-        public bool IsLightingActive => debugDisplaySettings.IsLightingActive;
-        public bool TryGetScreenClearColor(ref Color color) => debugDisplaySettings.TryGetScreenClearColor(ref color);
+
+        // All common settings are owned by another panel, so they are treated as inactive here.
+
+        public bool AreAnySettingsActive => false;
+        public bool IsPostProcessingAllowed => true;
+        public bool IsLightingActive => true;
```

Comment:
Feels a bit inconsistent that these properties are Upper Camel Case. But I guess this was already done before.

Reply:
Yeah the reason here was that I didn't want to go through deprecation process just to change the first letter to be lowercase, as this is implementing an interface that was already public (even though this & other implementing classes have been internal up until now). If you'd prefer consistency here, I can do it. What do you think?

Reply:
I personally think it's worth it as consistency decreases the WTF! moment users will get. 
If it's not too much trouble I would love to get that change but it can also be done in a followup PR.